src/api/app/policies/comment_policy.rb

```diff
@@ -0,0 +1,25 @@
+class CommentPolicy < ApplicationPolicy
+  def initialize(user, record)
```

Comment:
The reason that I'm overwriting this method is because we are using pundit helper in the views and  as the initialize method of `ApplicationPolicy` have `raise Pundit::NotAuthorizedError, "must be logged in" unless user` We couldn't realize any action If we are not logged, including see comments of a package